### DISCUSSION TITLE
Possible Override?

### DISCUSSION THREAD

**Discussion by PeteUnique** [Jason-Morcos/homebridge-magic-occupancy#30] [Ideas]
Loving the plug-in, is it possible to have a way to override the motion part. It would be handy if it is used in a bedroom, when asleep you don’t want motion triggering the lights. I use a watch to trigger sleep mode that could trigger a switch to disable it, then turn off the switch when sleep is turned off. So it’s just a way to stop it from working during the night

**Comment by Jason-Morcos** [comment]
Hey there! I would do this by installing the Dummy Switch plugin and creating a dummy switch that's something like "Disable motion Lights". Then, using a more complex automation setup app than the home app (ex. https://apps.apple.com/us/app/controller-for-homekit/id1198176727), you could set your automations related to the motion sensor to only run when that switch is on/off. Does that make sense?

This other issue may be helpful in this regard - https://github.com/Jason-Morcos/homebridge-magic-occupancy/issues/23
